tasks/Open To Evacuate or Not to Evacuate.md

```diff
@@ -0,0 +1,44 @@
+# Summary
+Subjects played a decision-making game in which the goal is to make an appropriate decision regarding whether to evacuate from an impending ‘disaster’ that would wipe out subjects' endowment unless they evacuated in time. 
+
+# References
+Main Paper: https://royalsocietypublishing.org/doi/full/10.1098/rspa.2019.0685#d1e503
+Data Files: https://humannaturelab.net/publications/collective-communication-and-behaviour-in-response-to-uncertain-danger-in-network-experiments
+
+# Stimuli
+## The visual components
+![image](https://user-images.githubusercontent.com/78745728/117169986-9b2cbf00-ad97-11eb-84d7-66849fa6f64d.png)
+
+
```

Comment:
Network topology varied as either a random-regular network or a ring-lattice network. Network size was also manipulated to have 10, 20, 40, or 60 nodes.